src/octokit-queries.ts

```diff
@@ -1,35 +1,37 @@
 import * as core from '@actions/core';
 import { GitHub } from '@actions/github/lib/utils';
+import { isString } from './functions';
 
 export const getTeamSlugsForAuthor = async (
   octokit: InstanceType<typeof GitHub>,
   org: string,
-  author: string,
+  username: string,
+  ignoreSlugs: string[] = [],
 ): Promise<string[]> => {
   const { data: allTeams } = await octokit.rest.teams.list({
     org,
   });
-  const teamSlugs = allTeams.map((team) => team.slug);
 
-  const authorsTeamSlugs = [];
+  const authorsTeamSlugs = allTeams.map<Promise<string | undefined>>(async ({ slug }) => {
+    if (ignoreSlugs.includes(slug)) {
+      return;
+    }
```

Comment:
Not a fan of returning undefined in a `map` (that's why I used `for ...  of` initially), I think it could be cleaner by chaining with a filter: `.filter(ignoredSlugs.include).map(...)`

Reply:
Sure, I can take another pass at this, hopefully sometime later this week.

Reply:
Appreciate it!

Reply:
Updated in 1b535c37d3f46ba4338be8ec6aec78d51b74104c.